public/style.css

```diff
@@ -14,4 +16,55 @@
 
 ul {
   list-style: none;
+  overflow: hidden;
+  justify-content: center;
+}
+
+li {
+  display: inline-block;
+  width: 30%;
```

Comment:
But I have margin-right on  li so that will  push the second option below if I set the width: 50%

Reply:
Then you can use `width: calc(50% - 5px)` I think

Reply:
I had some issues with it but finally, I could figure it out and implement it.
 I left some explanation on CSS for why I used width: calc (50% - 90px) instead of calc (50% - 5px), hope it makes sense .

Reply:
It would be better to give the `li` a padding of `1 rem` on all sides, and a margin-right of `1 rem`, then you can subtract `3 rem` of the 50% width: `calc(50% - 3rem)`, that's a bit more clear than a 'magical' 90px.

Then it might also be good to give the `ul` a `display: flex` and enable flex-wrap, then all the answers are the same size (in case the text is multi-line)

Reply:
You probably also need to subtract the 3px border of the answer element (twice, once for each side) so that when an answer gets selected the items don't jump around.